comment/utils.py

```diff
@@ -1,10 +1,11 @@
 from django.conf import settings
+from django.db.models import Q
 from django.core.mail import send_mail
 from django.contrib.auth.models import User
 
 
 def send_comment_to_admin_email(comment):
```

Comment:
It is a bit unclear what does `comment` parameter contain. Of course, from `CommentViews` we know that it contains the dict with serialized data, however it is not straightforward. 

As an option you can pass parameters from `CommentViews` like this:
```
utils.send_comment_to_admin_email(name=serializer.validated_data['name'], phone=serializer.validated_data['phone'], ...)
```
The tradeoff is that the function will have a lot of parameters which may have own disadvantages :)